eclass/user.eclass

```diff
@@ -1,684 +0,0 @@
-# Copyright 1999-2022 Gentoo Authors
-# Distributed under the terms of the GNU General Public License v2
-
-# @ECLASS: user.eclass
```

Comment:
You'll have to mark it `@DEAD` and give it formal last-rites on the mailing list for 30 days, but I guess this commit was just for CI.